src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -39,12 +54,36 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   scheduleStartHour,
   personalizedEvent: isPersonalizedEvent = false,
 }) => {
+  const eventRef: RefObject<HTMLDivElement> = useRef(null);
```

Comment:
Why do we need this `eventRef`?

Reply:
it's needed to `collapse` an `expanded` event when user clicks outside of the event on the schedule
info from here ( item #2 ) covers that case https://github.com/sparkletown/sparkle/pull/1733#discussion_r665959998

Reply:
This is not being used anywhere. You can remove it.

Reply:
removed ea55e53